internal-import-file/import-document-ai/src/requirements.txt

```diff
@@ -1,3 +1,5 @@
+antlr4-python3-runtime>=4.13,<4.15
```

Comment:
I can't reproduce the problem locally, is it the circular import you mentioned ? can we check after moving the import in `relations_allowed.py` ?